syncer/syncer_test.go

```diff
@@ -135,6 +135,32 @@ func (s *testSyncerSuite) resetMaster() {
 	s.db.Exec("reset master")
 }
 
+func (s *testSyncerSuite) catchUpBinlog() {
+	ch := make(chan interface{})
+	ctx, cancel := context.WithCancel(context.Background())
+
+	go func() {
+		for {
+			ev, _ := s.streamer.GetEvent(ctx)
+			if ev == nil {
+				return
+			}
+			ch <- struct{}{}
+		}
+	}()
+
+	for {
+		t := time.NewTimer(10 * time.Millisecond)
```

Comment:
is `10 millisecond` short?

Reply:
1. `BinlogStreamer` is reading relay log from background, which should be fast enough than this time interval.
2. This `s.catchUpBinlog` is commonly used when some tests meet error and leave some unread binlog out.
So I think 10ms is enough here